INTERPRETING.md

```diff
@@ -34,6 +34,20 @@ properties of the global scope prior to test execution.
 - **`print`** A function that exposes the string value of its first argument to
   the test runner. This is used as a communication mechanism for asynchronous
   tests (via the `async` flag, described below).
+- **`$`** An ordinary object with the following properties:
+  - **`createRealm`** - a function which creates a new [ECMAScript
+    Realm](http://www.ecma-international.org/ecma-262/6.0/#sec-code-realms),
+    defines this API on the new realm's global `this` value, and returns the
+    `$` property of the new realm's global `this` value
+  - **`detachArrayBuffer`** - a function which implements [the
+    DetachArrayBuffer abstract
+    operation](http://www.ecma-international.org/ecma-262/6.0/#sec-detacharraybuffer)
+  - **`evalScript`** - a function which accepts a string value as its first
```

Comment:
There are some details (e.g. redefinition of global lexical bindings) that
can't be simulated via `eval`, so I think there's value in requiring a
dedicated API for script evaluation.

As for the active realm record: I actually intended for the same realm to be
used in both cases: `otherRealm.$` should be the global "this" value of
`otherRealm`, and `otherRealm.evalScript` should evaluate the provided code in
`otherRealm`.

In writing, "the current Realm", I was hoping to avoid the awkwardness of "the
realm in which `$` was defined", instead relying on [the formal definition of
"the current realm"](https://tc39.github.io/ecma262/#current-realm) and the
execution context created by invoking the `evalScript` function. But maybe my
definitions aren't rigorous enough to rely on those definitions--the intended
value of the function's [[Realm]] slot likely isn't clear at all.

Do you have any suggestions on how I could make this more clear?